src/04-modality-specific-files/02-magnetoencephalography.md

```diff
@@ -378,7 +378,7 @@ actual anatomical nasion: `sub-0001_ses-001_acq-NAS_photo.jpg`
 
 ![placement of NAS fiducial](images/sub-0001_ses-001_acq-NAS_photo.jpg "placement of NAS fiducial")
 
-### 3-D head point /electrode locations
+## Head shape and electrode description (`*_headshape.<manufacturer_specific_extension>`)
```

Comment:
This heading is a tad too long https://travis-ci.com/bids-standard/bids-specification/builds/104761114#L483

Reply:
```suggestion
## Head shape and electrode description (`*_headshape.<ext>`)
```